node-manager/src/event.rs

```diff
@@ -326,15 +326,43 @@ impl EventHandler {
                     }
                 }
             }
-            Event::OpenChannelRequest { .. } => {
-                // Unreachable, we don't set manually_accept_inbound_channels
+            Event::OpenChannelRequest {
+                temporary_channel_id,
+                counterparty_node_id,
+                ..
+            } => {
                 self.logger.log(&Record::new(
                     lightning::util::logger::Level::Debug,
-                    format_args!("EVENT: OpenChannelRequest ignored"),
+                    format_args!("EVENT: OpenChannelRequest incoming"),
                     "event",
                     "",
                     0,
                 ));
+
+                match self.channel_manager.accept_inbound_channel(
+                    &temporary_channel_id,
+                    &counterparty_node_id,
+                    0, // we don't track our channels currently
```

Comment:
if we accept multiple channels will there be a problem with overlap, should we generate a random number here instead?

Reply:
I haven't ran into it yet but shouldn't be difficult to already do a random number, will add that.

Reply:
Added random number generation here.